notebooks/failure-type-classification/testgrid_flakiness_detection.ipynb

```diff
@@ -18,7 +18,7 @@
   },
```

Comment:
<p>This can be included in the top cell. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/284/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>